net/openssh/files/sshd.init

```diff
@@ -6,8 +6,99 @@ STOP=50
 
 USE_PROCD=1
 PROG=/usr/sbin/sshd
+NAME=sshd
 
-start_service() {
+. /lib/functions.sh
+. /lib/functions/network.sh
+
+validate_section_sshd()
+{
+	uci_load_validate sshd sshd "$1" "$2" \
+		'PasswordAuth:bool:1' \
+		'RootPasswordAuth:bool:1' \
+		'RootLogin:bool:1' \
+		'Interface:string' \
+		'Port:port:22' \
+		'IdleTimeout:uinteger:0' \
+		'MaxAuthTries:uinteger:3' \
+		'enable:bool:1' \
```

Comment:
`enable` -> `enabled`

Reply:
It was enable in dropbear so I though I should keep it enable here too.